src/universal_transfer_operator/data_providers/database/google/bigquery.py

```diff
@@ -155,6 +240,209 @@ def truncate_table(self, table):
         """Truncate table"""
         self.run_sql(f"TRUNCATE {self.get_table_qualified_name(table)}")
 
+    def is_native_autodetect_schema_available(self, file: File) -> bool:
+        """
+        Check if native auto-detection of schema is available.
+
+        :param file: File used to check the file type of to decide whether there is a native auto-detection available.
+        """
+        supported_config = self.NATIVE_AUTODETECT_SCHEMA_CONFIG.get(file.location.location_type)
+        if supported_config and file.type.name in supported_config["filetype"]:  # type: ignore
+            return True
+        return False
```

Comment:
nitpick: Not sure whether it's better this way (less code but not sure how everyone feels regarding readbility)

```suggestion
        return bool(supported_config and file.type.name in supported_config["filetype"])
```

Reply:
https://github.com/astronomer/apache-airflow-provider-transfers/pull/66/commits/e83030f053afc687621114a710444fdaf51b3a0c